tests/test_task.py

```diff
@@ -1238,21 +1242,24 @@ def test_return_value(self):
                     transient_run=True,
                     store_result=True,
                     args=(expected_value,),
-                    task_name=f'first re-run transient')
+                    task_name='first re-run transient')
                 value = value_task.get()
                 self.assertEqual(value, expected_value)
+                task_graph.close()
+                task_graph.join()
             else:
                 with self.assertRaises(RuntimeError):
                     value_task = task_graph.add_task(
                         func=_return_value_once,
                         transient_run=True,
                         store_result=True,
                         args=(expected_value,),
-                        task_name=f'expected error {iteration_id}')
+                        task_name='expected error {iteration_id}')

```

Comment:
```suggestion
                        task_name=f'expected error {iteration_id}')

```

Reply:
I think this still needs an `f`-string, no?

Reply:
Whoops, that was sloppy of me!  Thanks for catching that.